mine/models/intermediate/int_products.sql

```diff
@@ -0,0 +1,17 @@
+with products as ( select * from {{ref ('stg_products')}} )
+    ,product_names as (select
+      * from {{ref('stg_product_category_name_translation')}})
```

Comment:
The inconsistent line break placement is a bit challenging to read. To help make things more readable, we often also add some whitespace like what dbt Labs recommends [here](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md#ctes)